types/webpack-dev-server/package.json

```diff
@@ -2,6 +2,8 @@
   "private": true,
   "dependencies": {
     "chokidar": "^3.5.1",
-    "http-proxy-middleware": "^2.0.0"
+    "http-proxy-middleware": "^2.0.0",
+    "tapable": "2.2.1",
+    "webpack": "^5.54.0"
```

Comment:
Why is `webpack` added here?
In #43232 we are talking about how the dependency on `@types/webpack` causes issues, those are separate packages.

Also, the dependency on `@types/webpack` does not need to be such a high/specific version number. `"^5"` should be just fine, but even better perhaps, would be simply `"*"`, as the dev server types just add to the Webpack types.

Reply:
Maybe even better would be a peerDependency on `@types/webpack` for `"^4 | ^5"`, as I am unsure about earlier major verions.

However, that will require more work to be done when version 6 comes, and I don't know how DefinatelyTyped views peerDeps vs deps.

Do you have any thoughts on this, @lex111?

Reply:
it works with '*' and without `tapable`, @henrikhermansen thx!

Reply:
Awesome! Thanks for taking the time to fix this annoying issue 😃 Happy to help!